osbenchmark/benchmark.py

```diff
@@ -634,8 +635,18 @@ def add_workload_source(subparser):
         action="store_true",
         default=False)
 
-    for p in [list_parser, test_execution_parser, compare_parser, aggregate_parser, download_parser, install_parser,
-              start_parser, stop_parser, info_parser, create_workload_parser]:
+    auto_aggregate_parser = subparsers.add_parser("auto-aggregate",
+                                                  add_help=False,
+                                                  help="Run multiple test executions with the same configuration and aggregate the results",
+                                                  parents=[test_execution_parser])
+    auto_aggregate_parser.add_argument(
+        "--test-iterations",
+        type=int,
+        required=True,
+        help="Number of test executions to run and aggregate")
```

Comment:
Could `auto-aggregate` be renamed to `execute-tests` and `auto_aggregate_parser` be renamed to `multiple_test_executions_parser`? Those names would be more suitable since the main intention behind this subcommand is to allow users to run multiple tests. 

But before renaming them, please consider the following:
If all the parameters between the test_execution_parser and auto_aggregate_parser are the same except `--test-iterations`, what are your thoughts on omitting a separate subcommand and just incorporating `--test-iterations` into test_execution_parser and have the multiple tests logic performed in `elif subcommand == "execute-test"`? 

There doesn't seem to be a large enough difference between running a single test and running multiple tests that justifies creating another subcommand. But let me know if there's another reason why a separate subcommand might be better.

Reply:
Discussed offline and added in the revision 👍